### ISSUE TITLE
Ombi does not match Emby nor Sonarr

### ISSUE THREAD

**Issue by diedrichg** [tidusjar/Ombi#1093]
#### Ombi Version:

V 2.1.1404

#### Update Branch:

development

#### Operating System:

Win8.1

#### Mono Version (only if your not on windows)

(Place text here)

#### Applicable Logs (from `/logs/` directory or the Admin page):

```

(Logs go here. Don't remove the ``` tags for showing your logs correctly. Please make sure you remove any personal information from the logs)

```

#### Problem Description:

See here [https://emby.media/community/index.php?/topic/44267-ombi-for-emby/page-10#entry418697](url)

Ombi 2.1.1404 shows e11-15 as available, skips e16, and then shows e17 as available. Sonarr and Emby agree that the only media on the disk is e11 and e12. This was a general test of Ombi to see if it was showing the correct content from Emby. According to this, it's not. The checkboxes can not be removed indicating to me that it's showing what it thinks is on the disk even though the media doesn't exist except for s11, s12.

#### Reproduction Steps:

Test a season with content already on the server to see if Ombi picks up the existing episodes correctly.

**Comment by simonk83** [comment]
I'm new to Ombi so still getting to grips with it, but I'm seeing this as well.  I deleted an episode of a show from disk and refreshed Sonarr and Emby.  Both show that episode as missing.

When I go to request that show in Ombi and choose episodes, every episode is marked with a checkbox, including the missing one, and I can't uncheck anything.   I'm trying to force all the Emby caching to run in the Scheduled Jobs tab but it doesn't seem to be helping.   Next up is a shutdown and reopen of Ombi to see if that helps at all.

**Comment by SuperPotatoMen** [comment]
@simonk83 
You don't have the same problem. See : https://github.com/tidusjar/Ombi/issues/1102

**Comment by SuperPotatoMen** [comment]
Can you upgrade to the lasted version? we have fixed a couple of problems. maybe its solved for you now

**Comment by diedrichg** [comment]
It appears to be fixed in 2.2.1596. I will open a new ticket if I see this again.

**Comment by SuperPotatoMen** [comment]
Cool!
